c/iothub_client/tests/iothubclient_http_e2etests/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 #this is CMakeLists.txt for iothubclient_http_e2etests
 cmake_minimum_required(VERSION 2.8.11)
 
-if(NOT (${use_amqp} AND ${use_http}))
+if(NOT (${use_amqp} OR ${use_http}))
```

Comment:
@markrad - We need to have AMQP present even when running http E2E tests. The reason is that to grab the "events" from the EventHub behind IoTHub we only "talk" AMQP. I'll revert only that change before the merge, but everything else looks GOOD. You don't need to do anything on your side, but wanted to let you know I'm making that one change to your changelist. Thanks!